src/core/numerics/reconstructions/weno3.hpp

```diff
@@ -0,0 +1,86 @@
+#ifndef CORE_NUMERICS_RECONSTRUCTION_WENO3_HPP
+#define CORE_NUMERICS_RECONSTRUCTION_WENO3_HPP
+
+#include "core/data/vecfield/vecfield_component.hpp"
+#include "core/utilities/index/index.hpp"
+#include <utility>
+
+namespace PHARE::core
+{
+template<typename GridLayout, typename SlopeLimiter = void>
+class WENO3Reconstruction
```

Comment:
In the documentation of this class if could be nice to put the reference of the paper where all this can be found